ompy/vector.py

```diff
@@ -424,6 +424,67 @@ def to_keV(self) -> Vector:
         else:
             raise NotImplementedError("Units must be keV or MeV")
 
+    def closest(self, E: ndarray, side: Optional[str]='right',
+                inplace=False) -> Union[Vector, None]:
+        """ Returns a vector with the values are set by
+            E[i] <= E[j] < E[i+1], values[i].
```

Comment:
sorry, but I don't understand this. I struggle especially with the `values[i]` part. It is not clear to me what the function does

Reply:
Do I get it correctly that this returns a vector with the *new bins set to the (old) bin values closest in energy*?

Reply:
I wonder whether you could/should remove `E[i] <= E[j] < E[i+1]` here ,as you choose from the "side" which one to take?

Reply:
I've updated the docstring. It should be clearer what the function does.